src/olympia/abuse/templates/abuse/emails/reporter_takedown.txt

```diff
@@ -1,3 +1,5 @@
 {% extends "abuse/emails/base_reporter.txt" %}{% load i18n %}{% block content %}{# L10n: This is an email. Whitespace matters #}{% blocktranslate %}
-Thank you for your report about the following {{ type }} on Mozilla Add-ons: {{ name }}. We reviewed your report and the content in question, and found that it violated Mozilla's add-on policies, and have therefore taken action to remove the content.
+Thank you for your report about the following {{ type }} on Mozilla Add-ons: {{ name }}.
```

Comment:
Let's fix https://github.com/mozilla/addons-server/issues/21758 while we're at it

Reply:
what about the points raised in the issue?  (Not that I think 15 email templates rather than 3 is that practical...)

Reply:
Sorry, that's what I meant about fixing that issue: we need to add some `if/else` depending on the `type` to change the entire email, because as it stands it doesn't work at all in multiple languages.

Reply:
(If that's more practical maybe split the email into multiple strings to only have one string that is separate by type ?)